sources/packages/web/src/components/layouts/aest/AESTApplicationSideBar.vue

```diff
@@ -93,12 +89,12 @@ export default defineComponent({
           });
         },
       },
-      restrictionsManagement: {
-        label: "Restrictions Management",
+      applicationRestrictionsManagement: {
+        label: "Application Restrictions Management",
         icon: "mdi-close-circle-outline",
```

Comment:
Have we checked on the icon expectation here from the business?

Reply:
There is no icon expectation in the ticket and if there is any feedback, we can address that in the upcoming ticket and not here.

Reply:
This icon is similar to the cross icon in the Student Details page displaying the "Restrictions" tab along other tabs.
![image](https://github.com/user-attachments/assets/558cf644-5bc4-47d1-b27d-433f8846e97d)

Agree with @dheepak-aot on this point, and we can address it in the upcoming ticket.